querybook/server/logic/metastore.py

```diff
@@ -270,6 +271,9 @@ def create_table_information(
         hive_metastore_description=hive_metastore_description,
     )
 
+    if description:
+        new_table_information.description = description
```

Comment:
what if the description from metastore is an empty string ""? may want to check it by
`if description is not None`

Reply:
In Python empty strings are considered falsy, so this if statement should cover that case

Reply:
what I meant was if the description is an empty string from metastore, we would like to honor it and load the empty description to querybook to overwrite the old value.

Reply:
Good catch, yes that is a good use case.